pkgs/by-name/ju/just/package.nix

```
@@ -9,16 +9,22 @@
   libiconv,
   mdbook,
   nix-update-script,
+  installShellCompletions ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
+  installManPages ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
+  withDocumentation ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
```

```suggestion
  # run the compiled `just` to build the completions
  installShellCompletions ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
  # run the compiled `just` to build the man pages
  installManPages ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
  # run the compiled `generate-book` utility to prepare the files for mdbook
  withDocumentation ? stdenv.buildPlatform.canExecute stdenv.hostPlatform,
```

Just proposing some comments as initially someone (such as myself :sweat_smile: ) might think `mdbook` and `installManPage` are going to be executable on `buildPlatform` coming from `nativeBuildInputs` but it's more about using `just` or `generate-book`